modules/client/src/multisig/internal/client/encoding.ts

```
@@ -61,8 +63,8 @@ export class MultisigClientEncoding extends ClientCore
       ],
     );
     return {
-      id: LIVE_CONTRACTS[SupportedVersion.LATEST][networkName]
-        .multisigRepoAddress,
+      id: contracts[aragonNetwork][SupportedVersions.V1_3_0]?.MultisigRepoProxy
+        .address || "",
```

What happens in consuming code if the id is `""`?

---

I dont think there is a case when this returns "", it will happen if we decide to deploy a new version of the contracts but for some reason we decide not to deoply in certain network. We could a add a property in the configs repo where we keep track of which version is the latest one

---

> I dont think there is a case when this returns ""

If this should never happen, can we throw an error if it happens?

---

sure :)